tracer/build/_build/UpdateVendors/VendoredDependency.cs

```diff
@@ -63,6 +63,13 @@ static VendoredDependency()
                 pathToSrc: new[] { "dnlib-3.4.0", "src" },
                 transform: filePath => RewriteCsFileWithStandardTransform(filePath, originalNamespace: "dnlib"));
 
+
+            Add(
+                libraryName: "Datadog.Sketches",
+                version: "1.0.0",
+                downloadUrl: "https://github.com/DataDog/sketches-dotnet/archive/v1.0.0.zip",
+                pathToSrc: new[] { "sketches-dotnet-1.0.0", "src", "Datadog.Sketches" },
+                transform: filePath => RewriteCsFileWithStandardTransform(filePath, originalNamespace: "Datadog.Sketches", AddNullableDirectiveTransform));
```

Comment:
Can you add a quick comment to explain that we're using `AddNullableDirectiveTransform()` because `Datadog.Sketches` enabled nullable reference types in the project file? Took me a minute to figure this out.